### DISCUSSION TITLE
Ability to integrate with keymaster

### DISCUSSION THREAD

**Discussion by MuscleGeek69** [tsightler/ring-mqtt#731] [Feature Requests]
Hello,

Can someone integrate with Keymaster? I run an Airbnb and use the rental control integration to automate certain things when guests check in and out. I would love to be able to update the door codes if possible. 

Thank you!

**Comment by tsightler** [comment]
This project doesn't have any ability to manipulate any codes so it's not really clear what you are asking for.  When you say "door codes" do you mean like Z-wave locks?
